pkg/activitypub/store/memstore/memstore.go

```diff
@@ -208,11 +208,11 @@ func (s *activityStore) get(activityID string) (*vocab.ActivityType, error) {
 	return a, nil
 }
 
-func (s *activityStore) query(query *spi.Criteria, opts ...spi.QueryOpt) *activityIterator {
+func (s *activityStore) query(query *spi.Criteria, opts ...spi.QueryOpt) *ActivityIterator {
```

Comment:
Eventually (once the Aries memory store supports the additional query options) we wouldn't need to have a separate in-memory implementation in this `orb` repo